css/style.css

```diff
@@ -0,0 +1,928 @@
+@import url('https://fonts.googleapis.com/css2?family=Lora:wght@400;600&family=Roboto+Condensed:wght@400;700&display=swap');
+
+* { box-sizing: border-box; }
+
+:root {
+  --prime-color: #f72210;
+}
+
+div { font-family: 'Roboto Condensed', sans-serif; }
+
+a { color: black; }
+
+a:hover {
+  text-decoration: none;
+  color: black;
+}
+
+a.hover-underline:hover { border-bottom: 1px solid var(--prime-color); }
+
+a.hover-prime:hover { color: var(--prime-color); }
+
+a.hover-dark { background-color: #272727; }
+
+.page-container { max-width: 1309px; }
+
+.font-roboto { font-family: 'Roboto Condensed', sans-serif; }
+
+.font-arial { font-family: Arial, Helvetica, sans-serif; }
+
+
+/* ============================================== Nav Section ============================================= */
+
+nav {
+  position: sticky;
+  top: -3px;
+  z-index: 2;
+}
+
+.nav-top {
+  background-color: #f72210;
+  height: 50px;
+  align-items: center;
+}
+
+.nav-top .time-span { display: none; }
+
+.nav-logo {
+  width: 132px;
+  height: 20px;
+  margin-left: 1em;
+}
+
+.nav-top-right span {
+  color: white;
+  font-weight: 600;
+  font-size: 1.15rem;
+}
+
+.btn-subs {
+  color: var(--prime-color);
+  font-weight: 600;
+  font-size: 1.15rem;
+  background-color: white;
+  padding: 0.2em;
+}
+
+.btn-subs i { margin-left: 0.5em; }
+
+.btn-nav-menu {
+  background-color: transparent;
+  color: #fb9a94;
+  font-size: 1.8rem;
+  position: relative;
+  top: 4px;
+}
+
+.nav-bottom {
+  padding: 1em 0.2em;
+  padding-right: 158px;
+  position: relative;
+  border-bottom: 2px solid transparent;
+}
+
+.nav-bottom .nav-bottom-links {
+  display: flex;
+  flex-flow: row nowrap;
+  justify-content: center;
+  width: 100%;
+}
+
+.nav-bottom-links span {
+  display: inline-block;
+  white-space: nowrap;
+  flex-grow: 1;
+  padding: 0.2em 1px;
+  text-align: center;
+  border-right: 1px solid #d9d9d9;
+}
+
+.nav-bottom a {
+  font-size: 1.15rem;
+  font-weight: 700;
+}
+
+.nav-bottom-links a:hover { color: var(--prime-color); }
+
+.nav-bottom-right {
+  position: absolute;
+  top: 21px;
+  right: 4.6em;
+  color: rgba(0, 0, 0, 0.36);
+}
+
+.search-span {
+  font-size: 1.032rem;
+  margin-right: 0.3em;
+}
+
+.curvy-span {
+  position: absolute;
+  top: 0;
+  right: 0;
+  width: 80px;
+  border-left: 12px solid white;
+  border-top: 13px solid var(--prime-color);
+}
+
+/* =========================================== Main Contents Section ====================================== */
+
+.main-contents {
+  position: relative;
+  padding: 0;
+  margin-top: 2px;
```

Comment:
bootstrap classes:
`position-relative p-0 mt-3`

Reply:
This selection targets 28 elements.

Reply:
This is a bootstrap project, no discussion, no compromises.